src/taco-utils/tacoPackageLoader.ts

```diff
@@ -322,7 +322,7 @@ module TacoUtility {
         private static createTacoPackageInstallRequest(packageKey: string, dependencyConfigPath: string, logLevel: InstallLogLevel): IPackageInstallRequest {
             if (fs.existsSync(dependencyConfigPath)) {
                 try {
-                    var dependencyLookup: any = require(dependencyConfigPath);
+                    var dependencyLookup: any = JSON.parse(<any> fs.readFileSync(dependencyConfigPath));
```

Comment:
It does return a buffer, and JSON.parse can handle a buffer, but our typings for JSON.parse don't acknowledge that so it would complain. The code runs fine though.